src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -367,13 +373,17 @@ public ContentItemDisplay GetEmpty(string contentTypeAlias, int parentId)
         [OutgoingEditorModelEvent]
         public ContentItemDisplay GetEmptyByKey(Guid contentTypeKey, int parentId)
         {
-            var contentType = Services.ContentTypeService.Get(contentTypeKey);
-            if (contentType == null)
+            using (var scope = _scopeProvider.CreateScope())
             {
-                throw new HttpResponseException(HttpStatusCode.NotFound);
-            }
+                var contentType = Services.ContentTypeService.Get(contentTypeKey);
+                if (contentType == null)
+                {
+                    throw new HttpResponseException(HttpStatusCode.NotFound);
+                }
+                scope.Complete();
 
-            return GetEmpty(contentType, parentId);
+                return GetEmpty(contentType, parentId);
```

Comment:
I could be wrong, but I think we should call `scope.Complete()` after doing `GetEmpty` as far as I can see `GetEmpty` performs write operations through `Services.ContentService.Create` so if something goes wrong I think we want to have that rolled back.

Reply:
Good point!